src/parser/priest/shadow/modules/features/DarkThoughts.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+import Analyzer, { SELECTED_PLAYER, Options } from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+import SPELLS from 'common/SPELLS/index';
+import SpellLink from 'common/SpellLink';
+import Statistic from 'interface/statistics/Statistic';
+import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
+import BoringSpellValueText from 'interface/statistics/components/BoringSpellValueText';
+import { When, ThresholdStyle } from 'parser/core/ParseResults';
+import { t } from '@lingui/macro';
+import EventHistory from 'parser/shared/modules/EventHistory';
+
+// Example log: /report/JAPL1zpDfN7W8wck/33-Heroic+The+Council+of+Blood+-+Kill+(5:46)/Mayrim/standard/statistics
+class DarkThoughts extends Analyzer {
+  static dependencies = {
+    eventHistory: EventHistory,
+  };
+  protected eventHistory!: EventHistory;
+
+  procsGained: number = 0;
```

Comment:
you dont need to specify that they are numbers if you are assigning a value to it immediately, it will figure that out itself